<?xml version="1.0"?>
<launch>
	<include file="$(find gazebo_ros)/launch/empty_world.launch">
		<arg name="gui" value="true" />
	</include>

	<node name="part_spawner" pkg="bcr_bot" type="part_spawner.py" output="screen"/>

</launch>